services/queue/src/monitor.js

```diff
@@ -4,4 +4,46 @@ const manager = new MonitorManager({
   serviceName: 'queue',
 });
 
+manager.register({
+  name: 'resolvedQueuePoll',
+  type: 'resolved-queue-poll',
+  version: 1,
+  level: 'info',
+  description: 'Report result of polling messages from an azure queue.',
+  fields: {
+    messages: 'Number of messages fetched.',
+    failed: 'Number of these messages that failed to be handled.',
+    resolver: 'The name of the resolver that polled the queue.',
```

Comment:
Could this be the name of the queue?

Reply:
The resolver itself doesn't actually know the name of the queue it is pulling from. I could add it in if wanted?

Reply:
I think I was just suggesting a change to the description: "The name of the queue being polled".

Reply:
(it doesn't have to be the actual name of the Azure object)